frontend/components/sidebar/left/SidebarLeftIndex.vue

```diff
@@ -95,63 +95,63 @@ interface SidebarLeftSelectorType {
 
 const organizationButtons: SidebarLeftSelectorType[] = [
   {
-    btnText: "sidebar.about",
+    btnText: "components.sidebar.left.index.about",
```

Comment:
Maybe it does make sense to not include the full directory structure for all of this, as in this case we'd have `sidebar` and `left` in there multiple times. I'll think on this, but also for non-tech localizers it might make sense to make it as simple as possible.

Reply:
Its true.. What do you think should I put there for now?

Reply:
I'd say it'd be easier if we leave out the full directory structure. Components are named in the way that they are such that the name reflects the directory in which they're found :) If I were to see the following two examples I think I'd know they're the same, so let's keep it easy on ourselves and pick the short:

- `components.sidebar.left.sidebar-left-index.about`
- `components.sidebar-left-index.about`

If a person wanted to find it in the code, they should be able to do so easily with both versions. Consistency would be key though 😊

Reply:
Especially with `components` in the key, we'd definitely be able to know where it is :)